actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -172,7 +172,24 @@ pub fn call<BS: Blockstore, RT: Runtime<BS>>(
                 ),
 
                 CallKind::DelegateCall => {
-                    todo!()
+                    // first invoke GetBytecode to get the code CID from the target
```

Comment:
This comment says exactly what is visible in code i.e doesn't bring much value. I think a comment makes sense when it describes the "why" or some nuance which a reader can't understand based on the code only.

Reply:
It is subtle, that's why I left it there.